### PR TITLE
fixes #6568 - showing ovirt template version if applicable

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: app/models/concerns/fog_extensions/ovirt/template.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module FogExtensions
+  module Ovirt
+    module Template
+      def full_name
+        "#{name}#{full_version.present? ? " (#{full_version})" : nil}"
```
Comment (lzap) [inline app/models/concerns/fog_extensions/ovirt/template.rb]: I'd prefer complex coding outside of strings interpolation, but this is solely my opinion. :-)

**Comment by lzap** [comment]
The code looks good, let me know once the new gem is built in nightly repos to test this patch. Thank you.

File: app/models/concerns/fog_extensions/ovirt/template.rb (hunk 1 above)
Comment (unorthodoxgeek) [inline app/models/concerns/fog_extensions/ovirt/template.rb]: you're right, will fix.

**Comment by domcleal** [comment]
(Please don't rebase this until the gems are available, or the bot will continue to reset the status.)

**Comment by unorthodoxgeek** [comment]
@domcleal - fog and rvovirt released the gems, I updated the gemfiles. please review :)

File: bundler.d/ovirt.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'rbovirt', '~> 0.0.24'
+  gem 'rbovirt', '~> 0.0.28'
```
Comment (domcleal) [inline bundler.d/ovirt.rb]: .32 according to the tags on https://github.com/abenari/rbovirt/commit/e9292f5d8bfb4eb2b11c6323933ebf87349e479f

**Comment by unorthodoxgeek** [comment]
@eLobato @domcleal - this should be merge-able :smile:

**Comment by dLobatog** [comment]
@unorthodoxgeek Almost, I'll rebase your commits into one and it'll be ready to :ship:

**Comment by unorthodoxgeek** [comment]
@eLobato - done
image

**Comment by dLobatog** [comment]
Merged as a647442376aceae2e539791790eb8f308b56cd3a, thanks @unorthodoxgeek

**Pull request closed without merging**